cmd/check.go

```diff
@@ -13,3 +18,25 @@ var checkCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(checkCmd)
 }
+
+func checkJunitAndWrite(cfg runtime.Config, results runtime.Results) error {
+	if viper.GetBool("junit") {
+		cfg.ResponseFormat = "junitxml"
+		junitformatter, err := formatters.NewForConfig(cfg)
+		if err != nil {
+			return err
+		}
+		junitResults, err := junitformatter.Format(results)
+		if err != nil {
+			return err
+		}
+
+		junitFilename, err := artifacts.WriteFile("results-junit.xml", string(junitResults))
```

Comment:
SUPER NIT (sorry): Wasn't there an extension method built into the formatter that we should use here?

Reply:
Yes. However, it assumes the filename is "results.<ext>". I don't want to clobber the main results file. I'm ok with this one, as it 1) will always be ".xml", and 2) will always be junit.